python/GafferTest/NodeAlgoTest.py

```diff
@@ -43,6 +43,8 @@ class NodeAlgoTest( GafferTest.TestCase ) :
 
 	def testUserDefaults( self ) :
 
+		Gaffer.Metadata.deregisterValue( GafferTest.AddNode, "op1", "userDefault" )
```

Comment:
Can we move this to `tearDown`, and also deregister any other registrations made by this class?

Reply:
Addressed in 67d4edb. I rebased and dropped this commit.